CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+
+# Contribution Rules📚:
+
+- You are allowed to make pull requests. We just merge it ;)
+- Do NOT remove other content.
+- Styling/code can be pretty, ugly or stupid, big or small as long as it works
+- Try to keep pull requests small to minimize merge conflicts
```

Comment:
A PR can be big. It just needs to fit in one feature. So, I think rephrasing is needed.